### ISSUE TITLE
Headers / Src for installing modules into the kernel

### ISSUE THREAD

**Issue by michieal** [damentz/liquorix-package#88]
Is it possible to update the linux headers code so that things like VMWare Player will work? Often times we at Apophis Software have to use window programs for 3d modeling, but do everything else in linux. It would be really awesome to not have to reboot every time we need to switch between kernels.

Thank you.

**Comment by damentz** [comment]
Hi @michieal, does VMWare Player require different headers than VMWare Workstation?  I use VMWare Workstation with the latest version using Arch's AUR here: https://aur.archlinux.org/packages/vmware-workstation.  It works today on all my systems that have it installed with the current version of Liquorix.

Looking at the AUR itself, it appears that there are some patches being applied.   Perhaps you need to apply those same patches to VMWare Player?

**Comment by michieal** [comment]
To be honest, I am not sure. (Answering the question asked.) I do know that about a year or so ago, VMWare worked awesomely. Now, however, when I start it, the player comes up and says that it needs to install it's modules into the new kernel. That's when it breaks. It finds the header path and says "woohoo!" lol. Then it fails on stddef.h & stdarg.h. I doubled checked and those files are in their proper places, gcc is where it belongs, and the path settings for gcc's .h files work elsewhere.
I'll admit, it's been ages sense I worked in such low-level coding areas as a kernel, so I am kind of out of my league on this, though it's my job to figure out. If it will help I have also included the log file.

How do I rip apart an AUR, to see the patches, so that I can find/make them for Ubuntu?
[vmware-32667.log](https://github.com/damentz/liquorix-package/files/8343088/vmware-32667.log)
